index.html

```diff
@@ -0,0 +1,151 @@
+    </div>
+
+    <div class="date">
+        <h3>Join us on the 5th and 6th of January 2023</h3>
+        <p class="venue">@ Vivanta Taj Group of Hotels, Shillong</p>
+    </div>
+</section>
+
+
+    <section class="container">
+        <h3>
+            Main Program
+        </h3>
+        <hr>
+        <section class="container2">
+
+            <div class="wrap">
+                <i class="fa-brands fa-node"></i>
+                 <h3>Node JS</h3>
+            
+                <p>Node.js is a back-end JavaScript runtime environment. It runs on the V8 JavaScript Engine, and executes JavaScript code outside a web browser.</p>
+            </div>
+
+            <div class="wrap">
+                <i class="fa-brands fa-js"></i>
+                 <h3>JavaScript</h3>
+                 <p>JavaScript is the Programming Language for the Web. JavaScript can update and change both HTML and CSS. JavaScript can calculate, manipulate and validate data.</p>
+            </div>
+
+            <div class="wrap">
+                <i class="fa-solid fa-code-branch"></i>
+                 <h3>Grapghs</h3>
+                 <p>In computer science, a graph is an abstract data type that consists of finite set of vertices together with a set of edges that represent links between vertices.</p>
+            </div>
+
+            <div class="wrap">
+                <i class="fa-solid fa-cloud"></i>
+                 <h3>Cloud</h3>
+            
+                <p>The cloud" refers to servers that are accessed over the Internet, and the software and databases that run on those servers</p>
+            </div>
+
+            <div class="wrap">
+                <i class="fa-solid fa-database"></i>
+                 <h3>DataBase</h3>
+                 <p>A database is a repository of information managed by a database engine which ensures integrity of data and fast access to the data.</p>
+            </div>
+
+    </section>
+        <a class="linksj">SEE THE WHOLE PROGRAM</a>
+</section>
+
+
+<section class="speakers">
+    <h3>Featured Speakers</h3>
+    <hr>
+    <div class="speakers-a">
+    </div>
+</section>
+</div>
+
+
+<footer>
+    <div class="part">
+            <h2>Affiliate Groups</h2>
+            <hr>
+
+        <div class="part-team">
+                <h3><i class="fab fa-microsoft"></i>Microsoft</h3>    
+                <h3><i class="fab fa-google"></i>Google</h3>
+                <h3><i class="fa-brands fa-linux"></i>Linux</h3>
+                <h3><i class="fa-brands fa-apple"></i>Apple</h3>
+                <h3><i class="fa-brands fa-airbnb"></i>Airbnb</h3>
+        </div>  
+    </div>
```

Comment:
- [ ] The footer on the home page should have a dark background, kindly change it to be aligned with requirements.

Reply:
Thank you for the feedback.